packages/forms/signals/src/field/structure.ts

```diff
@@ -75,40 +90,231 @@ export abstract class FieldNodeStructure {
     return this._injector;
   }
 
-  constructor(
-    /** The logic to apply to this field. */
-    readonly logic: LogicNode,
-  ) {}
+  constructor(logic: LogicNode, node: FieldNode, createChildNode: ChildNodeCtor) {
```

Comment:
nit: Use parameter properties? 

```suggestion
  constructor(
    readonly logic: LogicNode,
    readonly node: FieldNode,
    readonly createChildNode: ChildNodeCtor,
  ) {}
```

Reply:
It's more just something I'm experimenting with, it's not part of our style guide (yet).